src/Test/RepositoryManager.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+namespace Doctrine\Bundle\PHPCRBundle\Test;
+
+use Doctrine\Bundle\PHPCRBundle\DataFixtures\PHPCRExecutor;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+use Doctrine\Common\DataFixtures\Loader;
+use Doctrine\Common\DataFixtures\ProxyReferenceRepository;
+use Doctrine\Common\DataFixtures\Purger\PHPCRPurger;
+use Doctrine\Common\Persistence\ManagerRegistry;
+use Doctrine\ODM\PHPCR\DocumentManagerInterface;
+use Symfony\Bridge\Doctrine\DataFixtures\ContainerAwareLoader;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * A helper class for tests to reset the repository and load fixtures.
+ */
+class RepositoryManager
```

Comment:
i will follow this up by refactoring symfony-cmf/testing to use this class if its available.